cpu/esp32/Makefile.include

```diff
@@ -7,9 +7,7 @@ endif
 
 ESP_SDK_DIR = $(ESP32_SDK_DIR)
 
```

Comment:
```suggestion

```
And remove the additional blank line. Unfortunately, multiline suggestions are not supported for deleted lines :worried: